client/src/utils/api.js

```diff
@@ -38,6 +38,13 @@ export async function getRiderImage({ alias }) {
   return data.picture;
 }
 
+export async function validateUser({ username, password }) {
+  const result = await fetch(
+    `/api/users?username=${username}&password=${password}`
+  );
+  return result;
+}
+
```

Comment:
don't send id and passwort in the url, because it could be tracked, send in the body
```suggestion
    `/api/user/login`, {
    method: 'POST',
    body: JSON.stringify({username, password),
    headers: {
        'Content-Type': 'application/json'
    });
  return result;
}

```

and take it from there on the server

````js
app.post('/api/user/login', (req, res) => {
    const { username, password } = req.body
....
});

Reply:
Was the original solution i wanted to take. But express did not allow me to do this. Got an error message saying, i can't use a req.body in GET-methods

Reply:
Ahhh, i tried to use a 'GET'-method with a body, which didn't work. Didn't think about taking a 'POST'-Route. Nice hint. Thx

Reply:
the method have to be a POST-Method

Reply:
i miss something you have to extend in your server

````js
app.use(bodyParser.json());
`````

Reply:
This is already included in `app.use(express.json())`